index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>News Summary</title>
+
+</head>
+<body onload="new NewsController().renderHeadlines()">
```

Comment:
Could we pull this out into an `interface.js`, similar to [the Thermostat challenge](https://github.com/makersacademy/thermostat-example/blob/master/src/interface.js)? 

This could also be a good place to implement the event listeners 🙂